tools/syz-testbed/html.go

```diff
@@ -387,6 +417,15 @@ href="?view={{$view.Name}}">█ {{$view.Name}}</a>
 		{{end}}
 	</tr>
 	{{end}}
+	{{if $uiTable.HasFooter}}
+		<tr>
+			<td>-</td>
+			{{range $c := $uiTable.Table.ColumnHeaders}}
+				<td>{{$uiTable.Table.GetFooterValue $c}}</td>
+			{{end}}
+		</tr>
+	{{end}}
+	</tbody>
```

Comment:
All this var is big enough to be moved into the external file.

Reply:
Updated the code to use `go:embed`.